examples/js/getting-started/README.md

```diff
@@ -0,0 +1,21 @@
+# Getting started demo
+
+_This project was generated with [create-instantsearch-app](https://github.com/algolia/instantsearch.js/tree/master/packages/create-instantsearch-app) by [Algolia](https://algolia.com)._
```

Comment:
```suggestion
_This project was generated with [create-instantsearch-app](https://github.com/algolia/instantsearch/tree/master/packages/create-instantsearch-app) by [Algolia](https://algolia.com)._
```

Reply:
I guess we need to update all the README.md from templates in `create-instantsearch-app.
I'll take care of that.